onebusaway-android/src/main/res/xml/app_tracker.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="ga_trackingId">UA-2423527-19</string>
+
+    <bool name="ga_autoActivityTracking">true</bool>
+    <bool name="ga_reportUncaughtExceptions">true</bool>
+
+    <item
+        name="ga_dispatchPeriod"
+        format="integer"
+        type="integer">120
+    </item>
+
+    <string name="ga_sampleFrequency">90</string>
+
+    <bool name="ga_anonymizeIp">true</bool>
+    <bool name="ga_dryRun">false</bool>
+
+    <string name="ga_sampleFrequency">100.0</string>
+    <integer name="ga_sessionTimeout">500</integer>
+
+    <screenName name="com.joulespersecond.seattlebusbot.SearchActivity.">
```

Comment:
I believe the extra period should be removed here, so it would be:
`<screenName name="com.joulespersecond.seattlebusbot.SearchActivity">`